<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionDialog</class>
 <widget class="QDialog" name="OptionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>uart option</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>font option</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="uartOptionPage">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Uart Option:</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Serial Port:</string>
              </property>
              <property name="buddy">
               <cstring>portNameComboBox</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="portNameComboBox"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Baud Rate:</string>
              </property>
              <property name="buddy">
               <cstring>baudRateComboBox</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="baudRateComboBox"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Date Bits:</string>
              </property>
              <property name="buddy">
               <cstring>dataBitsComboBox</cstring>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QComboBox" name="dataBitsComboBox"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Parity Type:</string>
              </property>
              <property name="buddy">
               <cstring>parityBitsComboBox</cstring>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QComboBox" name="parityBitsComboBox"/>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Stop Bits:</string>
              </property>
              <property name="buddy">
               <cstring>stopBitsComboBox</cstring>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QComboBox" name="stopBitsComboBox"/>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Flow Type:</string>
              </property>
              <property name="buddy">
               <cstring>flowTypeComboBox</cstring>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QComboBox" name="flowTypeComboBox"/>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Timeout:</string>
              </property>
              <property name="buddy">
               <cstring>timeoutLineEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QLineEdit" name="timeoutLineEdit"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="fontoptionPage">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>Font Style:</string>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <item row="1" column="0">
               <widget class="QCheckBox" name="fontBoldCheckBox">
                <property name="text">
                 <string>Bold</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Font Name:</string>
                </property>
                <property name="buddy">
                 <cstring>fontNameComboBox</cstring>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>109</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="0" column="1" colspan="4">
               <widget class="QFontComboBox" name="fontNameComboBox"/>
              </item>
              <item row="1" column="4">
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>Font Size:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="fontSizeComboBox">
                  <property name="minimumSize">
                   <size>
                    <width>50</width>
                    <height>0</height>
                   </size>
                  </property>
                  <item>
                   <property name="text">
                    <string>5</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>6</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>7</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>8</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>9</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>10</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>11</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>12</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>14</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>16</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>18</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>20</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>22</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>24</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>26</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>28</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="2" column="0">
               <widget class="QCheckBox" name="fontItalicCheckBox">
                <property name="text">
                 <string>Italic</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>156</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okPushButton">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyPushButton">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelPushButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>listWidget</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelPushButton</sender>
   <signal>clicked()</signal>
   <receiver>OptionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>581</x>
     <y>480</y>
    </hint>
    <hint type="destinationlabel">
     <x>592</x>
     <y>482</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
